src/wayland/shm/pool.rs

```diff
@@ -1,4 +1,4 @@
-#![forbid(unsafe_op_in_unsafe_fn)]
```

Comment:
forbid was causing a conflict somewhere inside of thread_local macro, so I changed it to deny